docs/source/developer_guide/guides/10_modular_pipeline_digital_fingerprinting.md

```diff
@@ -164,7 +164,7 @@ There are a number of modules that are used in both the training and inference p
 
 Source: `examples/digital_fingerprinting/production/morpheus/dfp/modules/dfp_preproc.py`
 
-The `dfp_preproc` module is a functional component within the Morpheus framework that combines multiple data filtering and processing pipeline modules related to inference and training. This module simplifies the pipeline by consolidating various modules into a single, cohesive unit. The `dfp_preproc` module offers configurability for parameters such as cache directory, timestamp column name, pre-filter options, batching options, user splitting options, and supported data loaders for different file types.
+The `dfp_preproc` module is a functional component within the Morpheus framework that combines multiple data filtering and processing pipeline modules related to inference and training. This module simplifies the pipeline by consolidating various modules into a single, cohesive unit. The `dfp_preproc` module allows for the configuration of parameters such as the cache directory, timestamp column name, pre-filter options, batching options, user splitting options, and supported data loaders for various file types.
```

Comment:
"allows" is typically avoided because it has the same connotation of permission that "may" has.  Maybe "The `dfp_preproc` module supports configuring parameters..."  A more significant rewrite would be to use second person point-of-view so the focus is on what the reader can accomplish rather than what the software can accomplish.